messages/off-chain-registry/message_manager.go

```diff
@@ -0,0 +1,105 @@
+package offchainregistry
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"github.com/ava-labs/awm-relayer/config"
+	"github.com/ava-labs/awm-relayer/vms"
+	teleporterregistry "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/upgrades/TeleporterRegistry"
+	"github.com/ethereum/go-ethereum/common"
+	"go.uber.org/zap"
+)
+
+var (
+	OffChainRegistrySourceAddress = common.HexToAddress("0x0000000000000000000000000000000000000000")
+)
+
+const (
+	addProtocolVersionGasLimit uint64 = 500_000
+)
+
+type messageManager struct {
+	logger             logging.Logger
+	destinationClients map[ids.ID]vms.DestinationClient
+	registryAddress    common.Address
+}
+
+func NewMessageManager(
+	logger logging.Logger,
+	messageProtocolAddress common.Address,
```

Comment:
This is unused. I don't think we need to keep the function signature for `NewMessageManager` the same, so I'd just remove it. If we do need to keep it for some reason, just rename it to `_`

Reply:
Good call, removed.